akka-docs/src/main/paradox/deploying.md

```diff
@@ -119,32 +129,65 @@ to see what this looks like in practice.
 
 For the JVM to run well in a Docker container, there are some general (not Akka specific) parameters that might need tuning:
 
-### Resource limits
+#### Resource limits
 
 Docker allows [constraining each containers' resource usage](https://docs.docker.com/config/containers/resource_constraints/).
 
-#### Memory
+##### Memory
 
 You may want to look into using [`-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap`](https://dzone.com/articles/running-a-jvm-in-a-container-without-getting-kille) options for your JVM later than 8u131, which makes it understand c-group memory limits. On JVM 10 and later, the `-XX:+UnlockExperimentalVMOptions` option is no longer needed.
 
-#### CPU
+##### CPU
 
 For multi-threaded applications such as the JVM, the CFS scheduler limits are an ill fit, because they will restrict
 the allowed CPU usage even when more CPU cycles are available from the host system. This means your application may be
 starved of CPU time, but your system appears idle.
 
 For this reason, it is best to avoid `--cpus` and `--cpu-quota` entirely, and instead specify relative container weights using `--cpu-shares` instead.
 
-## In Kubernetes
+### Deploying to Kubernetes
```

Comment:
Perhaps this should be even ####, as Kubernetes implies Docker?

Reply:
I was pondering the same, will update thanks.